counter_app/packaging/linux/com.blunix.git.currency_converter.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.blunix.git.currency_converter</id>
+  
+  <name>Currency Converter</name>
+  <summary>An app for converting currencies</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>MITNFA</project_license>
+  
+  <description>
+    <p>
+      An app for calculating how much a set amount of one currency is worth in another currency
+    </p>
+  </description>
+
+  <launchable type="desktop-id">com.blunix.git.currency_converter.desktop</launchable>
+  <developer_name>Blunix GmbH</developer_name>
+  <releases>
+    <release version="1.0.1" date="2024-03-29"/>
+  </releases>
+  <url type="homepage">https://git.blunix.com/miscellaneous/currency-converter</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://git.blunix.com/miscellaneous/currency-converter/-/raw/main/assets/screenshots/screenshot.png</image>
```

Comment:
this should not reference the main branch but a specific commit to be immutable.